charts/signalen-frontend/README.md

```diff
@@ -0,0 +1,23 @@
+# Signalen frontend Helm chart
+
+This Helm chart contains the Signalen web frontend.
+
+## Configuration
```

Comment:
These are not entirely up-to-date any more. Maybe we can just remove this section for now?